pkgs/by-name/dt/dtsfmt/package.nix

```
@@ -0,0 +1,28 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "dtsfmt";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "mskelton";
+    repo = "dtsfmt";
+    rev = "v${version}";
+    hash = "sha256-abNHciHBhJtRfkp7nqMtTw7xIqehPZsRXa6fVnPGeJw=";
+    fetchSubmodules = true;
```

```suggestion
    fetchSubmodules = true;
    hash = "sha256-abNHciHBhJtRfkp7nqMtTw7xIqehPZsRXa6fVnPGeJw=";
```
maybe reorder because submodules can change hash